appveyor-windows-build-go.yml

```diff
@@ -75,7 +75,7 @@ test_script:
     # Reactivate virtualenv before running tests
     - "venv\\Scripts\\activate"
     - "docker system prune -a -f"
-    - "pytest -vv tests/integration/buildcmd/test_build_cmd.py -k test_with_go"
+    - "pytest -vv tests/integration/buildcmd/test_build_cmd.py -k test_building_go"
```

Comment:
Good catch, we should maybe run all the classes:
```
- "pytest -vv tests/integration/buildcmd/test_build_cmd.py -k TestBuildCommand_Go_Modules"
- "pytest -vv tests/integration/buildcmd/test_build_cmd.py -k TestBuildCommand_Go_Modules_With_Specified_Architecture"
```

(like in `appveyor-windows-build-python.yml`)

Not sure we need this as we don't have tests to build inside a container and invoking `arm64` is not supported:
`docker run --rm --privileged multiarch/qemu-user-static --reset -p yes`

Reply:
multiarch/qemu-user-static shouldn't be needed for Windows machines.

Reply:
Updated with following lines;
```
- "pytest -vv tests/integration/buildcmd/test_build_cmd.py -k TestBuildCommand_Go_Modules"
- "pytest -vv tests/integration/buildcmd/test_build_cmd.py -k TestBuildCommand_Go_Modules_With_Specified_Architecture"
```